source/features/expand-all-collapsed-code.tsx

```diff
@@ -0,0 +1,45 @@
+import select from 'select-dom';
+import delegate, {DelegateEvent} from 'delegate-it';
+import features from '../libs/features';
+
+const expanderSelector = '.js-expand.directional-expander';
+
+// When diff table's childlist changes, select and click expand button to load more lines
```

Comment:
Comments should explain what the code does, instead of paraphrase the code line by line.

```suggestion
// Waits for the next loaded diff part and clicks on any additional "Expand" buttons it finds
```